kalite/control_panel/templates/control_panel/partials/_facility_table.html

```diff
@@ -8,6 +8,18 @@
             <small>
                 <span class="help-tooltip glyphicon glyphicon-question-sign" data-toggle="tooltip" data-placement="right" title='{% trans "A facility is a physical location where learners learn." %}'></span>
             </small>
+            {% if missing_meta %}
+            <small>
+                <span class = "help‐tooltip glyphicon
+                               glyphicon‐exclamation‐sign" style
```

Comment:
Let's avoid inline styles. Can probably use a bootstrap 'warning' class or similar to get the orange colour.